lib/API.jsx

```diff
@@ -327,6 +327,20 @@ export default function API(network, args) {
             return performPOSTRequest(commandName, parameters);
         },
 
+        /**
+         * Initiate a screen share request with a user.
+         *
+         * @param {object} parameters
+         * @param {Number} parameters.accountID
+         * @param {string} parameters.CallSid
```

Comment:
Also should be upper-case

```suggestion
         * @param {String} parameters.CallSid
```